Userland/Libraries/LibWeb/CSS/Parser/DeprecatedCSSParser.cpp

```diff
@@ -217,7 +217,7 @@ static bool takes_integer_value(CSS::PropertyID property_id)
     return property_id == CSS::PropertyID::ZIndex || property_id == CSS::PropertyID::FontWeight || property_id == CSS::PropertyID::Custom;
 }
 
-static StringView parse_custom_property_name(StringView value)
+static StringView parse_custom_property_name(const StringView& value)
```

Comment:
fwiw this copy does not really affect performance, passing a `StringView` is equivalent to passing two numeric values, a pointer and a length.
but making it `const StringView&` doesn't really hurt anything either.